erpnext/erpnext_integrations/taxjar_integration.py

```diff
@@ -122,7 +127,7 @@ def get_tax_data(doc):
 
 
 def set_sales_tax(doc, method):
-	if not TAXJAR_CALCULATE_TAX:
+	if not TAXJAR_CALCULATE_TAX or not TAXJAR_ENABLED:
```

Comment:
```suggestion
	if not TAXJAR_ENABLED:
		return
	if not TAXJAR_CALCULATE_TAX:
```

Reply:
Do I need to check TAXJAR_ENABLED separately, Instead of check  in or condition, or do I need to remove TAXJAR_ENABLED from here? Please confirm